omnibus/lib/symbols_inspectors.rb

```diff
@@ -74,6 +78,6 @@ def initialize(binary, &block)
 
   def inspect()
     log.info(self.class.name) { "Inspecting binary #{@binary}" }
-    @block.call(`go tool nm #{@binary}`)
```

Comment:
Why this change?

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/31004/commits/9b56d4ede78dac2a6e16330fc01c157c61064699